scripts/reboot

```diff
@@ -218,7 +230,11 @@ clear_warm_boot
 # Update the reboot cause file to reflect that user issued 'reboot' command
 # Upon next boot, the contents of this file will be used to determine the
 # cause of the previous reboot
```

Comment:
Might want to update comment describing the current change

Reply:
Comment updated in revised change set.